toshi-magento-plugin/Shipping/view/frontend/templates/toshi.phtml

```diff
@@ -0,0 +1,18 @@
+<?php
+$script = \Toshi\Shipping\Helper\Data::SCRIPT_SANDBOX;
```

Comment:
I think you can change the script value here to pull in the URL from `window.checkoutConfig.toshiScript;`
As that is defined in the custom config provider.  So it saves the need to check for the script URL!

Reply:
Hi @minesh93 thank you for looking into it. We tried this and looks like we will not get the window,checkoutConfig values at this stage, this not possible to handle the check via `window.checkoutConfig.toshiScript;`